### ISSUE TITLE
Deb packages fails on Ubuntu 16.04 with current Ettus PPA package

### ISSUE THREAD

**Issue by dkozel** [kb1vc/SoDaRadio#3]
I am sorry to have two different installation issues open at once, but I will help find the causes of both of these! I really like the project and am happy to debug. Unfortunately I'm trying to demo this tomorrow so will just move onto the next strategy for the moment.

I have UHD 3.11.0.0 installed using the Ettus PPA. My first guess is that your deb is built against an  earlier version of UHD (likely UHD 3.10.2.0) and this runtime error is caused by the ABI changing.

The UHD PPA doesn't currently install library `.so` files with enough versioning numbers to stop this problem from happening. I'll install the earlier UHD and report if this issue goes away. If it does then the quickest answer is to add a note that the current `.deb` file is built for `UHD 3.10.2.x`

```
[dkozel@moscato Desktop] sudo apt install ./SoDaRadio-5.0.0-1.x86_64.Ubuntu-16.04.deb 
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Note, selecting 'sodaradio' instead of './SoDaRadio-5.0.0-1.x86_64.Ubuntu-16.04.deb'
The following NEW packages will be installed:
  sodaradio
0 upgraded, 1 newly installed, 0 to remove and 0 not upgraded.
Need to get 0 B/4,555 kB of archives.
After this operation, 12.9 MB of additional disk space will be used.
Get:1 /home/dkozel/Desktop/SoDaRadio-5.0.0-1.x86_64.Ubuntu-16.04.deb sodaradio amd64 5.0.0 [4,555 kB]
Selecting previously unselected package sodaradio.
(Reading database ... 336768 files and directories currently installed.)
Preparing to unpack .../SoDaRadio-5.0.0-1.x86_64.Ubuntu-16.04.deb ...
Unpacking sodaradio (5.0.0) ...
Setting up sodaradio (5.0.0) ...
[dkozel@moscato Desktop] SoDaRadio 
/usr/bin/SoDaServer: symbol lookup error: /usr/bin/SoDaServer: undefined symbol: _ZN3uhd3msg16register_handlerERKPFvNS0_6type_tERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEE

"Waited 155 seconds for socket file [/home/dkozel/.local/share/SoDaRadio/SoDa___cmd] to be created.  Is the radio process dead?"
```

**Comment by dkozel** [comment]
Ah! Launchpad automatically deletes older releases when they're superseded by a new one. The 3.10.2.0 packages are no longer available.

I installed 3.10.2.0 manually to `/usr` and was able to install and run the `SoDaRadio-5.0.0-1.x86_64.Ubuntu-16.04.deb` with no problem.

I did have to force the installation of the `.deb` because apt didn't know that UHD was installed.
```
sudo dpkg --force-all -i SoDaRadio-5.0.0-1.x86_64.Ubuntu-16.04.deb
```

**Comment by kb1vc** [comment]
I haven't built a new deb package in a while, as I moved my own work entirely to fedora.  I'll fire up a VM or two this weekend and rebuild the packages.  Though, I'm glad you got the older version to work.  

Do you have what you need for the demo?

**Comment by kb1vc** [comment]
This should be fixed -- there are new Ubuntu debs for 16.04 and 17.10 on the [main page at] (https://kb1vc.github.io/SoDaRadio/)
